spec/ddtrace/contrib/active_support/notifications/subscriber_spec.rb

```diff
@@ -11,6 +11,10 @@
       end
     end
 
+    before :each do
+      test_class.unsubscribe_all
```

Comment:
Ah, so this isn't supposed to work. It's not supposed to work because it's supposed to be `private`. But it works because this is a class method, and the `private` modifier doesn't work.

You could use [`#private_class_method`](http://ruby-doc.org/core-1.9.3/Module.html#method-i-private_class_method) or `class << self` syntax to address this.

Reply:
Sorry for that, I was having trouble running tests locally so I used the CI to test changes, and they apparently were broken. But I didn't have time to address this yesterday.